styles.css

```diff
@@ -444,3 +444,8 @@ body {
   cursor: pointer;
   border: none;
 }
+
+.get-in-touch:hover {
+  background: #0d0f5b;
+  color: #fff;
+}
```

Comment:
- [ ] As stated by the previous reviewer, please add pressed interaction when clicking on the button like this:
```
.get-in-touch:active {
  color: #2230D2;
}
```
- [ ] Also consider using correct color code for hover interaction (`#6070FF`)